packages/integration-sdk-cli/src/__tests__/cli-run-failure.test.ts

```diff
@@ -37,14 +38,7 @@ beforeEach(() => {
   process.env.JUPITERONE_ACCOUNT = 'mochi';
   loadProjectStructure('validationFailure');
 
-  polly = new Polly('run-cli-failure', {
```

Comment:
Small bonus change. Extracted this functionality into `createTestPolly` because it is used in 3 tests.